.circleci/config.yml

```diff
@@ -85,14 +85,14 @@ jobs:
   linting:
     <<: *common
     docker:
-      - image: circleci/python:3.6
+      - image: circleci/python:3.9

```

Comment:
This was the first thing I tried, but don't see why we shouldn't leave it on 3.9